mkdocs.yml

```diff
@@ -17,8 +17,8 @@ extra_css:
 extra_javascript:
   - js/asciinema-player-2.6.1.js
 repo_name: EasyBuild @ GitHub
-repo_url: https://github.com/easybuilders/easybuild-docs
-edit_uri: easybuild-docs/edit/main/docs
+repo_url: https://github.com/easybuilders/easybuild
```

Comment:
Let's use `easybuild-easyconfigs` repo here, since that has way more forks (but fewer stars)?

Reply:
done in 33f8a48